Framework/FormAdd.php

```diff
@@ -90,12 +90,12 @@ public function addLabel($name, $values){
      * @param array $choicesid List of choices Ids
      * @param array $values List of default values
      */
-    public function addSelect($name, $label, $choices, $choicesid, $values = array()) {
+    public function addSelect($name, $label, $choices, $choicesid, $values = array(), $isMandatory=false) {
```

Comment:
there is a addSelectManadatory, why adding new field here?

Reply:
there is a Form::addSelectMandatory(), not a FormAdd::addSelectMandatory() :/.
FormAdd is used in forms where  user can add lines (like admins list in space configuration)